common/client-libs/validator-client/src/validator_api/routes.rs

```diff
@@ -7,5 +7,7 @@ pub const API_VERSION: &str = VALIDATOR_API_VERSION;
 pub const MIXNODES: &str = "mixnodes";
 pub const GATEWAYS: &str = "gateways";
 
+pub const ACTIVE: &str = "active";
```

Comment:
Would it make sense to keep this info in an enum? Even if it only had one variant now, we might want to introduce further state variants in the future...